main.go

```diff
@@ -109,6 +114,7 @@ func main() {
 			setupLog.Error(err, "unable to create webhook", "webhook", "Observability")
 			os.Exit(1)
 		}
+		checkForWebhookServerReady(mgr)
```

Comment:
the error returned from the function needs to be checked. If the webhook times out, you can use the same `os.Exit(1)` to terminate the operator.